project/Common.scala

```diff
@@ -57,6 +57,7 @@ object Common extends AutoPlugin {
     Compile / doc / scalacOptions --= Seq("-Xfatal-warnings"),
     scalafmtOnCompile := true,
     autoAPIMappings := true,
+    apiURL := Some(url(s"https://doc.akka.io/api/akka-persistence-cassandra/${version.value}")),
```

Comment:
You may use `projectInfoVersion` here which is "snapshot" or the version.

Reply:
thanks, changed to that